tracer/src/Datadog.Trace/Sampling/GlobalSamplingRule.cs

```diff
@@ -18,25 +20,26 @@ public GlobalSamplingRule(float rate)
             _globalRate = rate;
         }
 
-        public string RuleName => "global-rate-rule";
```

Comment:
Were these rate values names used or defined anywhere previously?  🤔

Reply:
They are not part of the sampling rules RFC and they only showed up in debug logs.